bentoml/_internal/bento/build_config.py

```diff
@@ -515,15 +761,49 @@ def _converter(value: t.Union[OptionsCls, t.Dict[str, t.Any]]) -> options_type:
 
 @attr.define(frozen=True, on_setattr=None)
 class BentoBuildConfig:
-    """This class is intended for modeling the bentofile.yaml file where user will
-    provide all the options for building a Bento. All optional build options should be
-    default to None so it knows which fields are NOT SET by the user provided config,
-    which makes it possible to omitted unset fields when writing a BentoBuildOptions to
+    """
+    This class is intended for modeling the :code:`bentofile.yaml` file where user will
+    provide all the options for building a Bento.
+
+    All optional build options should be default to :code:`None` so BentoML knows
+    which fields are **NOT SET** by the user provided config.
+
+    Thus, it is possible to omitted unset fields when writing a :code:`BentoBuildConfig` to
     a yaml file for future use. This also applies to nested options such as the
-    DockerOptions class and the PythonOptions class.
+    :class:`DockerOptions` class and the :class:`PythonOptions` class.
+
+    Args:
+
+        service (:code:`str`):
+            The required name of the service to be built.
```

Comment:
this is not the service name
```suggestion
            The import_str of the target bentoml.Service to build
```